### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by olls** [itsapi/pycraft#3]
It would be nice if we could get it running on most platforms. I sorta got it running on Windows, the main hurdle to get it running was input, then it is playable. The issues we have are:
- [x] ~~The Windows input is only blocking which means we can't use arrow keys on the menu, and the game only runs while the user is inputting.~~
  Fixed with 48b162926785036766063dc76bb832de42daa2b6 - non blocking input works on windows.
- [x] ~~There is a problem with msvcrt.getch returning bytes instead of a string, this causes problems with arrow keys.~~
  Fixed with 3a0df67f0dcce9e49f0679ef096acd004dedbb1d - fix for Windows arrow keys input.
- [ ] Colours are the only real problem although it is playable without them.
- [ ] In Windows the clear screen code (and all escape codes...) doesn't work, ~~and arrow keys don't work~~ (No longer used). Add support for turning them off.

**Comment by grit96** [comment]
None of the escape codes will work on Windows. We should probably add support for turning off escape codes.

**Comment by olls** [comment]
![py](https://cloud.githubusercontent.com/assets/923336/11299267/575f7076-8f7d-11e5-9c3b-bf38cead00c1.png)

**Comment by olls** [comment]
Works well under Cygwin

**Comment by grit96** [comment]
:+1: nbinput and escape codes (incl colours)?

**Comment by grit96** [comment]
The other problem with Windows is that the terminal is too narrow and can't be made wider.

**Comment by olls** [comment]
Yes, only problems were with arrow keys and some Unicode characters

**Comment by olls** [comment]
https://github.com/adoxa/ansicon converts ANSI escape codes to the Windows functions.

Or we could do it manually: https://msdn.microsoft.com/en-us/library/windows/desktop/ms682088(v=vs.85).aspx#_win32_character_attributes (More fun!)
